pkg/model/firewall.go

```diff
@@ -205,6 +212,13 @@ func (b *FirewallModelBuilder) applyNodeToMasterBlockSpecificPorts(c *fi.ModelBu
 		protocols = append(protocols, ProtocolIPIP)
 	}
 
+	if b.Cluster.Spec.Networking.Romana != nil {
+		// Romana needs to access etcd
+		glog.Warningf("Opening etcd port on masters for access from the nodes, for romana.  This is unsafe in untrusted environments.")
+		tcpRanges = []portRange{{From: 1, To: 4001}, {From: 4003, To: 65535}}
```

Comment:
This is inconsistent with the above opened ports (e.g. 9600 is missing)

Reply:
This was based on what another CNI does. Port 9600 is covered by the second range (4003-65535).
I'm OK with making this more specific in both places.